pkg/backends/newrelic/newrelic.go

```diff
@@ -303,10 +328,23 @@ func NewClientFromViper(v *viper.Viper) (gostatsd.Backend, error) {
 	nr.SetDefault("enable-http2", defaultEnableHttp2)
 	nr.SetDefault("user-agent", defaultUserAgent)
 
+	//New Relic Config Defaults & Recommendations
+	v.SetDefault("statser-type", "null")
+	v.SetDefault("flush-interval", "10s")
+	if v.GetDuration("flush-interval").Seconds() < 10 {
+		log.Warnf("[%s] flush-interval below 10s, recommended to be >= 10s", BackendName)
+	} else if v.GetDuration("flush-interval").Seconds() == 10 {
+		log.Infof("[%s] flush-interval default 10s", BackendName)
+	}
+	if v.GetString("statser-type") == "null" {
+		log.Infof("[%s] internal metrics OFF, to enable set 'statser-type' to StatserInternal", BackendName)
```

Comment:
The internal statser type is `internal` (actually it's the default switch case, but "internal" is the intended type :)), maybe something more like "to enable set 'statser-type' to 'logging' or 'internal'"?

Reply:
oh yes, of course thats much better 👍